### DISCUSSION TITLE
How to mark a task instance as complete or skip it without modify the DAG, so the dag can move forward?

### DISCUSSION THREAD

**Discussion by aflyflyv2** [apache/dolphinscheduler#9848] [Ideas]
能否增加强制将某个任务设置成成功或者跳过执行？
有时候任务执行时，希望能够跳过 DAG 内某个任务（或者将它置为成功），从而推动 DAG 向后继续执行

**Comment by Tandoy** [comment]
Maybe you can set the sql task node like this

![Snipaste_2022-04-30_10-39-52](https://user-images.githubusercontent.com/56899730/166087300-ec39c77b-38c9-43f0-bfdf-19bbd2a5ccf4.PNG)

**Reply by aflyflyv2** [reply]
I know this way, but not convenient enough, you need to modify the dag.... (and restore it after the task complete,  too risky if forget to restore)
